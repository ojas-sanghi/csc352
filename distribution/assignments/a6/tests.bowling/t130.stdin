5
5
6
4
6
2
9
1
8
2
10
5
4
7
3
10
8
2
5
