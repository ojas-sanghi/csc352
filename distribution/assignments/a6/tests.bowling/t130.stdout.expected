Frame  1: ( 5/ 5) 16  16 ( Spare)
Frame  2: ( 6/ 4) 16  32 ( Spare)
Frame  3: ( 6/ 2)  8  40 (  Open)
Frame  4: ( 9/ 1) 18  58 ( Spare)
Frame  5: ( 8/ 2) 20  78 ( Spare)
Frame  6: (10/ -) 19  97 (Strike)
Frame  7: ( 5/ 4)  9 106 (  Open)
Frame  8: ( 7/ 3) 20 126 ( Spare)
Frame  9: (10/ -) 20 146 (Strike)
Frame 10: ( 8/ 2) 15 161 ( Spare) (+5)
